### PR TITLE
"Scroll down" arrow in homepage menu is misaligned #273

### PR DESCRIPTION
**Pull request by akshayvinchurkar**
This will fixed the arrow placement

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -3282,4 +3282,4 @@ article.panel-placeholder {
 	.twentyseventeen-front-page .entry-content blockquote.alignright {
 		margin-right: -20%;
 	}
-}
\ No newline at end of file
+}
```
Comment (ntwb) [inline style.css:3285]: Please, don't remove the newline from the end of the file

**Review by ntwb — CHANGES_REQUESTED** [review]

File: style.css (hunk 1 above)
Comment (akshayvinchurkar) [reply style.css:3285]: i did not even touch the code i just make 0.76em to 0.40em and thats it😒 how this is possible😕

Comment (melchoyce) [reply style.css:3285]: Maybe it's something your code editor does automatically?

**Comment by davidakennedy** [comment]
Fixed here: https://github.com/WordPress/twentyseventeen/commit/71430139d7bb1046a30b628b4f2094461274e1ef

**Pull request merged**